netlify/functions/subscribe.js

```diff
@@ -0,0 +1,36 @@
+const https = require('https')
+
+exports.handler = async function(event, context) {
+    const { email } = JSON.parse(event.body)
+
+    const data = {
+        email: email,
+        oid: "00D1U00000110AJ"
```

Comment:
I *think* we probably want to store this in the `.env` file and pass it to the netlify function when we call it from the gatsby UI. That'll require changes to the build setup in netlify.toml